apps/tad/base/deployment-pgadmin.yaml

```diff
@@ -12,15 +12,21 @@
       automountServiceAccountToken: false
       securityContext:
         fsGroup: 5050
-      initContainers:
-        - name: init-clean-sessions
+      containers:
+        - name: clean-sessions
```

Comment:
I wonder if this will fix the problems. Stated here:
https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-fault-recovery
Only the container giving a problem is restarted, so not the other container. I can not find a good solution.

An alternative approach could be to have the side container run the 'clean' script at a regular interval, and remove (session) files that are X-time old (like 24 hours orso?).

Could it also be related to permissions? Like mention here (uid=5050?):
https://github.com/helm/charts/issues/19690
It is very strange PGAdmin would only create sessions indefinitely and not remove them and thus eventually crash.

Reply:
I already fixes those..

Reply:
i created an init container to set the permissions as a test.